.bumpversion.cfg

```diff
@@ -0,0 +1,19 @@
+[bumpversion]
+current_version = 0.4.3.dev0
+commit = false
+tag = false
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<release>[a-z]+)(?P<dev>\d+))?
+serialize = 
+	{major}.{minor}.{patch}.{release}{dev}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
+optional_value = gamma
+values = 
+	dev
+	gamma
+
+[bumpversion:file:setup.py]
+
+[bumpversion:file:sphinx_rtd_theme/__init__.py]
+
```

Comment:
Perhaps `package.json` as well then? I know we don't package anything there at the moment, but maybe a good place to keep the version number updated either way

Reply:
Oh, also the version number in the docs could be a static string updated with this tool.

Reply:
Oops, forgot `package.json`, now added.

Also added the `conf.py` of the documentation, there is no version number in the documentation body text.